python/mxnet/ndarray.py

```diff
@@ -560,24 +559,6 @@ def wait_to_read(self):
         check_call(_LIB.MXNDArrayWaitToRead(self.handle))
 
 
-    @property
-    def shape(self):
```

Comment:
If the interface is stable, that's fine. I just want to make sure user doesn't make the wrong function call from SparseND which is directed to NDArray and doesn't work